e2e/specs/lab-orders.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { expect } from '@playwright/test';
+import { type Visit } from '@openmrs/esm-framework';
+import { generateRandomPatient, type Patient, startVisit, endVisit, deletePatient } from '../commands';
+import { test } from '../core';
+import { ChartPage, VisitsPage } from '../pages';
+
+let patient: Patient;
+let visit: Visit;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+  visit = await startVisit(api, patient.uuid);
+});
+
+test('Record, edit and delete a lab order', async ({ page }) => {
+  const patientChartPage = new ChartPage(page);
+  const visitsPage = new VisitsPage(page);
+
+  await test.step('When I visit the patient chart page', async () => {
+    await patientChartPage.goTo(patient.uuid);
+  });
+
+  await test.step('And I click on the `Clinical forms` menu', async () => {
+    await patientChartPage.page.getByLabel(/clinical forms/i).click();
+  });
+
+  await test.step('And I click on the `Laboratory Test Orders` link', async () => {
+    await page.getByText(/laboratory test orders/i).click();
+  });
+
+  await test.step('And I select a lab test', async () => {
+    await page.getByRole('button', { name: 'Add', exact: true }).click();
+    await page.locator('#tab select').selectOption('857AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA');
+  });
```

Comment:
Rather than using a tag with value of option here, can we do something with getByLabel()?

Reply:
Unfortunately, these form fields don't have their labels associated with the select inputs. So using `getByLabel` won't work here, @Vijaykv5.